index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
         </li>
       </ul>
     </section>
+    <section class="appendix informative">
+      <h2>Understanding Proof Sets and Proof Chains</h2>
+      <p>
+Proof sets and chains allow multiple proofs to be carried in a VC, i.e., instead
+of a single <em>proof</em> included in the signed VC we can express multiple
+proofs in an array as shown in the previous examples. The elements of this array
+are members of <em>proof set</em> and, optionally, a <em>proof chain</em>.
+The purpose of this section is to explain the intended use of each of these entities
+and, in particular, their differing security properties. These differing security
+properties lead to differences in the processing in section <a href="#add-proof-set-chain"></a>.
+      </p>
+      <p>
+We are going to represent VCs and their proofs in a highly abbreviated manner so
+that the key security properties can be observed.
+      </p>
+      <p>
+We consider a scenario with three signatories: a CEO, a CFO, and a VP of
+Engineering. Each will need to have a public and private key pair for signing a
+document. We denote the private/public keys of each of these signatories by
+<em>priv/pubCEO</em>, <em>priv/pubCFO</em>, and <em>priv/pubVPE</em>
+respectively.
+      </p>
+      <p>
+When constructing a <em>proof set</em> where each of the signatories signs a
+document without concern we construct a proof symbolically as:
+      </p>
+      <pre><code class="language-javascript">
+{
+  &quot;type&quot;: &quot;DataIntegrityProof&quot;,
+  &quot;cryptosuite&quot;: &quot;jcs-eddsa-2022&quot;,
+  &quot;created&quot;: &quot;2023-03-05T19:23:24Z&quot;,
+  &quot;proofPurpose&quot;: &quot;assertionMethod&quot;,
+  &quot;verificationMethod&quot;: pubCEO,
+  &quot;proofValue&quot;: signature(privCEO, document)
+}
+      </code></pre>
+      <p>
+Where <em>pubCEO</em> is used as a placeholder for a reference that resolves to
+the CEO's public key and <code>signature(priv, doc)</code> denotes the
+computation of a digital signature by a particular data integrity cryptosuite
+using a particular private key over a particular document. The <em>type</em>,
+<em>cryptosuite</em>,  <em>created</em>, and <em>proofPurpose</em> attributes do
+not factor into our discussion so we will omit them. In particular below we show
+all the proofs in a <em>proof set</em> on a document that has been signed by the
+VP of engineering, CFO, and CEO:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will verify
+  &quot;proof&quot;: [
+  {&quot;verificationMethod&quot;: pubVPE, &quot;proofValue&quot;: signature(privVPE, document)},
+  {&quot;verificationMethod&quot;: pubCFO, &quot;proofValue&quot;: signature(privCFO, document)},
+  {&quot;verificationMethod&quot;: pubCEO, &quot;proofValue&quot;: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+The <em>holder</em> or any other intermediary receiving a signed VC containing a
```

Comment:
Avoid abbreviations (such as VC) when at all possible, we are trying to ensure that the document uses linked terminology everywhere (where we spell out the world and link the term to its definition).